docs/learning/quantity/index.md

```diff
@@ -43,6 +45,14 @@ For a detailed description of all the setting supported by FormatProp see the EC
 
 ### Concepts
 
+#### Formats Provider
+
+A [FormatsProvider]($ecschema-metadata) interface helps provide all the necessary formats for displaying formatted quantity values, while also enabling users to add formats of their own.
+
+A [MutableFormatsProvider]($ecschema-metadata) interface extends the read-only FormatsProvider above by allowing adding or removing Formats to the provider.
+
+The [SchemaFormatsProvider]($ecschema-metadata) takes in a [SchemaContext]($ecschema-metadata), to provide Formats coming from schemas. The SchemaFormatsProvider also requires a [UnitSystemKey]($quantity) passed in to filter the formatProps returned, according to the current unit system set in the SchemaFormatsProvider.
```

Comment:
1. When referencing an API, it should be either a link (e.g. [SchemaFormatsProvider]($ecschema-metadata)), or in backticks (e.g. `SchemaFormatsProvider`).
2. In this section, "format" is used in consistently - in some places it starts with uppercase, in some others - not.
3. Terms like "formatProps" should be avoided. If this is a type, then it should be referenced appropriately (see 1st point). If it's a concept, then we should use general language.

Reply:
I've went through the whole document, and modified things such that each section, when mentioning a API for the first time, would use a hyperlink. Subsequent mentions of those APIs would use backticks, so should be good